src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl

```diff
@@ -773,6 +781,74 @@
           }
         }
       }
+    },
+    "/users/{userId}/stats": {
+      "get": {
+        "operationId": "getUserStats",
```

Comment:
please add tag (useful for code code gen)

Reply:
Did it!